model-optimizer/extensions/ops/proposal.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, graph: Graph, attrs: dict):
             'post_nms_topn': 300,  # default in caffe-shared
             'infer': ProposalOp.proposal_infer,
             'in_ports_count': 3,
-            'out_ports_count': 2,
+            'out_ports_count': 1 if attrs.get('version') == 'opset1' else 2,
```

Comment:
Can it be equal `opset2` where it is also 1? I think we need to be careful with it.

Reply:
@rkazants No, it can not. We have only Proposal-1 and Proposal-4 in IR's (I also checked it in the documentation)

Reply:
I propose to change to: 1 if attrs.get('version') < 'opset4' else 2

Reply:
No. We may have `version='experimental'` and in this case we should set 2 outputs

Reply:
Also, MO operate with specific versions of the operations where they were introduced and do not support "alias" names, so the current implementation is the best one.

Reply:
Ok.